config.xml

```diff
@@ -36,6 +36,5 @@
     </platform>
     <plugin name="cordova-plugin-whitelist" spec="^1.3.3" />
     <plugin name="cordova-plugin-chromecast" spec="https://github.com/jellyfin/cordova-plugin-chromecast" />
-    <plugin name="org.jellyfin.mobile" spec="src/NativeShell" />
```

Comment:
why?

Reply:
Oh dev-link-setup thing

Reply:
how can I turn that off?

Reply:
No idea, you can revert the change to that file for now though if it's not needed.

Reply:
reverted it

Reply:
it was part of that symlink to edit in android studio thing